lib/constructs/dynamodb/dynamodb-table.ts

```diff
@@ -1,29 +1,62 @@
-import { Construct, CfnOutput } from "@aws-cdk/core";
+import { CfnOutput, Stack } from "@aws-cdk/core";
 import { Table, TableProps, BillingMode, AttributeType } from "@aws-cdk/aws-dynamodb";
 import { App } from "@serverless-stack/resources";
 
 export type DynamoDBTableProps = TableProps
 
-export class DynamoDBTable extends Construct {
-	constructor(scope: App, id: string, tableName: string, partitionKey: string, props?: DynamoDBTableProps) {
+interface DynamoDbTableDefinition {
+  tableName: string;
+  partitionKey: string;
+}
+
+const DYNAMO_TABLES: Array<DynamoDbTableDefinition> = [
+  { tableName: 'BusinessUser', partitionKey: 'uid' },
+  { tableName: 'Business', partitionKey: 'place_id' },
+  { tableName: 'Redirect', partitionKey: 'unique_id' },
+  { tableName: 'SubscriberUser', partitionKey: 'mobile_number' },
+  { tableName: 'PushMessage', partitionKey: 'uid' },
+
+];
+
+export class DynamoDBTable extends Stack {
+  scope: App;
+	constructor(scope: App, id: string) {
     super(scope, id);
-    
-    const app: any = this.node.root;
-
-    const table = new Table(this, "Table", {
-      tableName: tableName,
-      billingMode: BillingMode.PAY_PER_REQUEST, // Use on-demand billing mode
-      partitionKey: { name: partitionKey, type: AttributeType.STRING },
-    });
-    
-    // Output values
-    new CfnOutput(this, "TableName", {
-      value: table.tableName,
-      exportName: app.logicalPrefixedName("TableName"),
-    });
-    new CfnOutput(this, "TableArn", {
-      value: table.tableArn,
-      exportName: app.logicalPrefixedName("TableArn"),
-    });
+    this.scope = scope;
+    console.log(scope.stageName);
+    console.log(scope.stage);
+    console.log(scope.region);
+    this.createDynamoTables(DYNAMO_TABLES);
+  }
+
+  createDynamoTables(tables: Array<DynamoDbTableDefinition>) {
+    for (const table of  tables) {
+      const { tableName, partitionKey } = table;
+
+      this.createTable(`dev-${tableName}`, partitionKey);
+      this.createTable(`prod-${tableName}`, partitionKey);
+    }
+  }
+
+  createTable(tableName: string, partitionKey: string) {
+    try {
+      const table = new Table(this, tableName, {
```

Comment:
We dont have to now but we may want to consider adding Cloudwatch alarms for some of the metrics associated with our tables